src/components/SøknadsSteg/2-VelgBarn/Barnekort/Barnekort.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Normaltekst, Element } from 'nav-frontend-typografi';
+import { Knapp } from 'nav-frontend-knapper';
+import { EtikettSuksess } from 'nav-frontend-etiketter';
+import barn1 from '../../../../assets/barn1.svg';
+import barn2 from '../../../../assets/barn2.svg';
+import barn3 from '../../../../assets/barn3.svg';
+import { useApp } from '../../../../context/AppContext';
+import { IBarn, ISøknadsfelt } from '../../../../typer/søknad';
+import { hentTilfeldigElement } from '../../../../utils/hjelpefunksjoner';
+
+const Barnekort: React.FC<IBarn> = ({ navn, ident, alder, borMedSøker, medISøknad }) => {
+    const { søknad, settSøknad } = useApp();
+    const ikoner = [barn1, barn2, barn3];
+
+    function settMedISøknad(erMed: boolean) {
+        settSøknad({
+            ...søknad,
+            barn: søknad.barn.map(barn =>
+                barn.ident === ident
+                    ? { ...barn, medISøknad: { ...barn.medISøknad, verdi: erMed } }
+                    : barn
+            ),
+        });
+    }
+
+    function fjernBarnFraSøknad() {
+        settMedISøknad(false);
+    }
+
+    function leggTilBarnISøknad() {
+        settMedISøknad(true);
+    }
+
+    return (
+        <div className="barnekort">
+            <div className="barnekort__header">
+                <img alt="barn" className="barneikon" src={hentTilfeldigElement(ikoner)} />
```

Comment:
Nice, men kanskje den burde hete `hentTilfeldigBarneikon` eller lignende? Litt pirkete nå 🙃 
Med mindre vi skal bruke den til flere ting, da er `hentTilfeldigListeElement` eller bare det den heter nå like greit, for er jo egentlig det selve funksjonen gjør 👌

Reply:
Brukte det først, men siden den kan brukes til arrays så la jeg den i utils hvis vi skulle trenge noe lignende senere 👌